### DISCUSSION TITLE
Under System Configuration CMP Configuration UI is blank in keyfactor/ejbca-ce image tag version 8.2.0.1

### DISCUSSION THREAD

**Discussion by NeerajGahlawatJava** [Keyfactor/ejbca-ce#506] [Q&A]
I am using keyfactor/ejbca-ce image tag version 8.2.0.1 . Under System Configuration CMP Configuration UI is blank Please find image attached for your ref.
<img width="716" alt="EJBCA_8 2 0 1" src="https://github.com/Keyfactor/ejbca-ce/assets/82153856/5290d530-ac7f-4a40-be71-640952930e42">

<img width="678" alt="addCMPaliasNotAvailable" src="https://github.com/Keyfactor/ejbca-ce/assets/82153856/5be5105a-b674-4d56-b5a4-50cb15e25aa0">

Is that expected if yes then how to configure  cmp .

Regards
Neeraj gahlawat

**Comment by primetomas** [comment]
Known issue. See here: https://github.com/Keyfactor/ejbca-ce/issues/473

**Reply by NeerajGahlawatJava** [reply]
Thanks @primetomas. Any expected date of next release of EJBCA with fix.

**Reply by primetomas** [reply]
This is currently planned for 8.3, in may unfortunately. I can see if it can be expidited.
